src/main/java/Alice.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Alice {
+    public static void main(String[] args) {
+
+        // Create scanner object to read input
+        Scanner scanner = new Scanner(System.in);
+
+        // Create an array to store the tasks entered by the user
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        // Create a variable to store the number of tasks so far
+        int taskNum = 0;
+
+        // Create string to store line separating responses
+        String line = "____________________________________________________________";
```

Comment:
good that you made your line into a variable so that code below becomes more readable